### PR TITLE
Features

### PR DESCRIPTION
**Pull request by danilowoz**
### Architecture

- [x] Typescript supports;
- [x] One config file for site-wide settings (`/content/site-config.js`);
- [x] React Context for global UI state, with SSR;
- [x] UI Folder with common components;
- [x] No index files: `Header/Header.tsx` and `Header/Header.css.ts`
- [x] Support alias and absolutes imports: `@` to access `./src`;

### SEO

- [x] SEO component: Social / meta tags;'
- [x] Generate sitemap automatically;
- [x] Generates all favicons for Web, Android, iOS,

### Style

- [x] [`styled-components`](https://www.styled-components.com/) v4
- [x] Reset.css;
- [x] Generated media queries for easy use;

### Quality code and others

- [x] Fomat Prettier by Lint stage (only update what is in HEAD);
- [x] [`size-plugin`](https://github.com/GoogleChromeLabs/size-plugin) to keep an eye on your bundle sizes
- [x] commitlint: Lint commit messages;
- [x] Eslint;
- [x] Run develop mode from Local LAN (0.0.0.0:8000);
- [x] Test support

### CODE REVIEW

**Comment by danilowoz** [comment]
- [x] TSLint by create react app
- [x] Reset.css
- [x] Breakpoints (mobile first)
- [x] Alias with no `@`
- [x] Remove Netlify
- [x] Check dependencies
- [x] Check gatsby ssr

File: content/site-config.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = {
     "digital",
     "product",
   ],
-  siteUrl: "{{= appUrl }}",
+  siteUrl: "http://www.google.com",
```
Comment (pdrbrnd) [inline content/site-config.js]: Is this correct?

Comment (danilowoz) [reply content/site-config.js]: No :(

File: package.json [hunk 2]
```diff
@@ -1,73 +1,72 @@
...
-    "gatsby-transformer-sharp": "^2.1.19",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-helmet": "^5.2.1",
     "styled-components": "^4.2.0",
-    "wipe.css": "^1.2.0"
+    "wipe.css": "^1.1.1"
   },
   "devDependencies": {
+    "@babel/code-frame": "^7.0.0",
```
Comment (pdrbrnd) [inline package.json]: do we need this one?

Comment (danilowoz) [reply package.json]: Test needs it

Comment (pdrbrnd) [reply package.json]: strange. I have jest in Significa.co and didn't need code-frame

Comment (danilowoz) [reply package.json]: You're right, it is not required. I was just trying to solve a mistake in alias things

**Review by pdrbrnd — APPROVED** [review]

**Pull request merged**